packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/BasicSecurity/TwoStepVerification/template.success.js

```diff
@@ -257,7 +277,10 @@ const TwoStepVerification = props => {
             defaultMessage='Two-Step Verification - Mobile Phone Number'
           />
           <Link size='14px' onClick={handleGoBack}>
-            Change
+            <FormattedMessage
+              id='scenes.security.twostepverification.cancel'
+              defaultMessage='Cancel'
+            />
```

Comment:
We can't have duplicate ids with the same default message.

Reply:
fixed